arduino/splitflap/esp32/splitflap_task.cpp

```diff
@@ -200,6 +204,13 @@ void SplitflapTask::runUpdate() {
             break;
         }
       }
+
+      if (recv_buffer_ready) {
```

Comment:
There's probably a better way to unify the serial parsing and mqtt message code, but this was quick and easy.